app/src/main/java/fr/free/nrw/commons/explore/map/ExploreMapMediaPresenterIml.kt

```diff
@@ -0,0 +1,23 @@
+package fr.free.nrw.commons.explore.map
+
+import dagger.Provides
+import fr.free.nrw.commons.Media
+import fr.free.nrw.commons.di.CommonsApplicationModule
+import fr.free.nrw.commons.explore.paging.BasePagingMapPresenter
+import fr.free.nrw.commons.explore.paging.PagingContract
+import io.reactivex.Scheduler
+import javax.inject.Inject
+import javax.inject.Named
+
+interface ExploreMapMediaPresenter : PagingContract.Presenter<Media>
+
+/**
+ * Presenter for ExploreMapFragment
+ */
+class ExploreMapMediaPresenterIml @Inject constructor(
+    @Named(CommonsApplicationModule.MAIN_THREAD) mainThreadScheduler: Scheduler,
+    dataSourceFactory: ExploreMapMediaDataSource,
+    isFromSearchActivity: Boolean
```

Comment:
This dependency isn't defined so the dagger does not know how to inject it here.